### ISSUE TITLE
Sponsor doc not working

### ISSUE THREAD

**Issue by nathenharvey** [cfgmgmtcamp/cfgmgmtcamp.github.io#323]
@toshywoshy the sponsor doc does not work for me.

Just me or does http://cfgmgmtcamp.eu/files/ConfigManagementCampGent2018SponsorDoc.pdf not contain a valid PDF?

**Comment by purpleidea** [comment]
The link returns text which points to:
https://git.cfgmgmtcamp.be/2018ghent/sponsordoc/-/jobs/31/artifacts/raw/sponsordoc.pdf
It's true this confuses the browser though. Probably a good idea to
update the link.

There's still no mgmtconfig in the communities list, but I guess we're
not real enough yet :)

Cheers!

**Comment by toshywoshy** [comment]
@nathenharvey It was not just you, depending on your pdf viewer, it might not open the redirection, in this case mine actually did so, and I did not realize there was a redirection in there, the point of your PR/issue was to change the location and make the 2 systems independent, a redirect does not do that, so I changed the gitlab => github pipeline to do that now correctly and copy the actual content of the pdf file.

@purpleidea
1) We do not want a redirection, that was @nathenharvey original point in his PR/issue.
2) As for your mgmtconfig, I added your community request.
